company-kata-solutions/src/test/java/org/eclipse/collections/companykata/Exercise8Test.java

```diff
@@ -60,7 +62,8 @@ public void totalOrderValuesByCity()
     {
         Function0<Double> zeroValueFactory = () -> 0.0;
         Function2<Double, Customer, Double> aggregator = (result, customer) -> result + customer.getTotalOrderValue();
-        MutableMap<String, Double> map = null;
+        MutableMap<String, Double> map = this.company.getCustomers()
+                .aggregateBy(Customer::getCity, zeroValueFactory, aggregator);
```

Comment:
Hi Nikhil, a couple of additional comments (sorry for the delay) - could you please format these chained method calls to separate lines?

Reply:
Done